spec/mailers/tasks_mailer_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe TasksMailer, type: :mailer do
+  describe "tasks_completed" do
+    let!(:mail) { TasksMailer.tasks_completed }
```

Comment:
TasksMailer.task_completed(watcher, @task, current_user).deliver

it's different from what I see in controller